src/Compilers/CSharp/Test/Semantic/Semantics/SemanticErrorTests.cs

```diff
@@ -12883,6 +12883,90 @@ void M(ref int i)
         }
 
         [Fact]
+        public void CS1510ERR_RefLvalueExpected_02()
+        {
+            var text =
+@"class C
+{
+    void M()
+    {
+        var a = new System.Action<int>(ref x => x = 1);
+        var b = new System.Action<int, int>(ref (x,y) => x = 1);
+        var c = new System.Action<int>(ref delegate (int x) {x = 1;});
+    }
+}
+";
+            DiagnosticsUtils.VerifyErrorsAndGetCompilationWithMscorlib(text,
```

Comment:
@leppie I am not following. How is that test is different? It creates a compilation and verifies diagnostics. It uses better (more readable, easier to maintain) style for the baseline.
```
                 // (7,25): error CS1511: Keyword 'base' is not available in a static method
                 //     static object FLD = base.ToString();
                 Diagnostic(ErrorCode.ERR_BaseInStaticMeth, "base"),
                 // (6,28): error CS1511: Keyword 'base' is not available in a static method
                 //     static CLS() { var x = base.ToString(); }
                 Diagnostic(ErrorCode.ERR_BaseInStaticMeth, "base"),
                 // (8,39): error CS1511: Keyword 'base' is not available in a static method
                 //     static object PROP { get { return base.ToString(); } }
                 Diagnostic(ErrorCode.ERR_BaseInStaticMeth, "base"),
                 // (9,37): error CS1511: Keyword 'base' is not available in a static method
                 //     static object METHOD() { return base.ToString(); }
                 Diagnostic(ErrorCode.ERR_BaseInStaticMeth, "base"),
                 // (14,19): warning CS0649: Field 'A.P' is never assigned to, and will always have its default value null
                 //     public object P;
                 Diagnostic(ErrorCode.WRN_UnassignedInternalField, "P").WithArguments("A.P", "null")
```

That baseline is even automatically generated by the test infrastructure when the test fails.